### PR TITLE
LPS-193625, LPS-193633 | Test Automation | Master

### PR DESCRIPTION
**Pull request by andreefarias**
Tickets:
* [LPS-193625](https://liferay.atlassian.net/browse/LPS-193625)
* [LPS-193633](https://liferay.atlassian.net/browse/LPS-193633)

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase [hunk 1]
```diff
@@ -22,6 +22,74 @@ definition {
 		}
 	}
 
+	@description = "LPS-149508 - Verify CRUD functionality with audit when using object definition"
+	@priority = 5
+	test AssertCRUDWithObjectDefinition {
+		property portal.acceptance = "true";
+
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase:28]: Let's not include this as an acceptance test until we're sure of the stability of the test.

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase [hunk 2]
```diff
@@ -22,6 +22,74 @@ definition {
...
+		task ("Given a custom object definition created") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase:29]: `Given a custom object definition`

Remember that adjectives go before the noun in English, so the correct phrasing would be `created object definition`

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase [hunk 3]
```diff
@@ -22,6 +22,74 @@ definition {
...
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 193625");
+
+			ObjectAdmin.editCustomObject(fieldLabelName = "New object definition updated");
+		}
+
+		task ("and Given a custom object definition deleted") {
+			ObjectAdmin.deleteObjectViaAPI(objectName = "CustomObject193625");
+		}
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase:46]: Combine this into one task:

`When the user updates and then deletes the object definition`

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase [hunk 4]
```diff
@@ -22,6 +22,74 @@ definition {
...
+
+		task ("When accessing the audit admin") {
+			Audit.openAuditAdmin();
+		}
+
+		task ("Then assert that the audit can see the custom object definition created, updated and deleted") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase:52]: Combine this into one task:

`Then the Add, Update, and Delete events appear in the Audit records`

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase [hunk 5]
```diff
@@ -22,6 +22,74 @@ definition {
 		}
 	}
 
+	@description = "LPS-149508 - Verify CRUD functionality with audit when using object definition"
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase:24]: Verify CRUD events appear in the Audit records for object definitions

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase [hunk 6]
```diff
@@ -90,6 +90,90 @@ definition {
 		}
 	}
 
+	@description = "LPS-164582 - Verify CRUD functionality with audit when using object definition entry that has the entry history activated"
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase:92]: Verify CRUD events appear in the Audit records for object entries when entry history is enabled

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase [hunk 7]
```diff
@@ -22,6 +22,74 @@ definition {
 		}
 	}
 
+	@description = "LPS-149508 - Verify CRUD functionality with audit when using object definition"
+	@priority = 5
+	test AssertCRUDWithObjectDefinition {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase:26]: AssertCRUDEventsWithObjectDefinition

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase [hunk 8]
```diff
@@ -90,6 +90,90 @@ definition {
 		}
 	}
 
+	@description = "LPS-164582 - Verify CRUD functionality with audit when using object definition entry that has the entry history activated"
+	@priority = 5
+	test AssertCRUDWithObjectEntry {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase:94]: AssertCRUDEventsWithObjectEntry

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase [hunk 9]
```diff
@@ -90,6 +90,90 @@ definition {
...
+		property portal.acceptance = "true";
+
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase:96]: Same thing here too, let's not add this to acceptance until we're sure it's stable

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase [hunk 10]
```diff
@@ -90,6 +90,90 @@ definition {
...
+		task ("Given a custom object definition with toogle for Entry History in Audit Framework activated and a custom field") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase:97]: Given a custom object definition with a text field and Entry History enabled

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase [hunk 11]
```diff
@@ -90,6 +90,90 @@ definition {
...
+				entry = "New object entry",
+				updateEntry = "New object entry updated");
+		}
+
+		task ("and Given a custom object entry deleted") {
+			ObjectAdmin.deleteObjectEntryViaAPI(
+				fieldName = "customField",
+				objectName = "CustomObject193633",
+				value = "New object entry updated");
+		}
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase:140]: Combine this:

`When the user creates, updates, and then deletes an object entry`

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase [hunk 12]
```diff
@@ -90,6 +90,90 @@ definition {
...
+
+		task ("When accessing the audit admin") {
+			Audit.openAuditAdmin();
+		}
+
+		task ("Then assert that the audit can see the custom object entry created, updated and deleted") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase:146]: Combine this:

`Then the Add, Update, and Delete events appear in the Audit records`

**Review by timpak — CHANGES_REQUESTED** [review]
@andreefarias please see my comments below. Once we are certain these tests are stable (passing consistently for an entire week), please go back and delete the ignored tests. You can create a ticket to for this task so we don't forget.

Once it's stable for a month or so, we can explore adding the new tests to acceptance.

**Comment by andreefarias** [comment]
ci:test:sf

**Pull request closed without merging**